docs/optimization_guide/nncf/ptq/basic_quantization_flow.md

```diff
@@ -150,12 +149,20 @@ Tune quantization parameters
        regex = '.*layer_.*'
        nncf.quantize(model, dataset, ignored_scope=nncf.IgnoredScope(patterns=regex))
 
+* ``target_device`` - defines the target device, the specificity of which will be taken into account during optimization. The following values are supported: ``ANY`` (default), ``CPU``, ``CPU_SPR``, ``GPU``, and ``VPU``.
+
+* ``advanced_parameters`` - used to specify advanced quantization parameters for fine-tuning the quantization algorithm. Defined by ``nncf.AdvancedQuantizationParameters`` class. ``None`` is default.
```

Comment:
What needs to be added here?

Reply:
I'm wondering if there is a reference to AdvancedQuantizationParamaters? It feels like something is missing, the docs say we can add this, that None is the default, but not what the possible values are.

Reply:
Right, we discussed internally whether we should describe it on OV level and decided that it's too advanced. I plan to add a cross-reference to NNCF class description from a new NNCF API documentation.

Reply:
added the reference